javascript/pickware-modern-rules.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable quote-props */
+
+module.exports = {
+    'plugins': [
+        'promise',
+    ],
+    'rules': {
+        'promise/always-return': 'error',
+        'promise/no-return-wrap': 'error',
+        'promise/param-names': 'error',
+        'promise/catch-or-return': 'error',
+        'promise/no-native': 'off',
+        'promise/no-nesting': 'error',
+        'promise/no-promise-in-callback': 'error',
+        'promise/no-callback-in-promise': 'error',
+        'promise/avoid-new': 'error',
```

Comment:
We currently cannot enable this because there is no `Promisify` in Shopware Next.
This code would not be allowed:
```js
            await new Promise((resolve, reject) => {
                documentFileWindow.addEventListener('load', resolve);
                documentFileWindow.addEventListener('error', reject);
            });
            documentFileWindow.print();
```

Reply:
Yep, then we should probably disable this rule in `shopware-next-plugin.js`. I'd keep it for the node projects since `new Promise` often leads to callback-style programming when promises should be used instead.